src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1294,6 +1295,7 @@
   "entity.fisherman.messagewatertoofar": "My hut seems to be too far away from water, and I can't find any water to fish in! It should be an area of at least 7x7 blocks, and the same height as my hut.",
   "com.minecolonies.coremod.furnaceuser.nofuel": "Please tell me what kind of fuel I should use in my furnace!",
   "com.minecolonies.coremod.furnaceuser.noore": "Please tell me what kind of ore I should smelt in my furnace!",
+  "com.minecolonies.coremod.furnaceuser.nofood": "Please tell me what kind of food I should hand out!",
```

Comment:
okay, then this message is just a bit confusing. It should rather be sth around "Please allow me to cook some more options, at this moment I can't provide any food for the colony" or so @ravenbuilder934 might have an idea.

Reply:
You could probably get away with "Please tell me what I should cook!" tbh, or a slightly more verbose "I can't provide food unless you tell me what to cook!"

Reply:
The problem is that the player "disabled all options" manually and not "never enabled any options"

Reply:
The statement was to answer the latter, and does fit both. But I suppose you could be even more verbose: "I can't provide any food because you have told me not to cook anything!", or less role-play: "Please re-enable some food options so I can cook something!"

Reply:
Yes, those two are fine for me

Reply:
"None of my enabled food options are in my restaurant!" maybe? Am I understanding this wrong?

Reply:
@Raycoms i already commented here, why'd you ask for another review? :P

Reply:
No, it's "All food in the game is disabled in the UI" that's when this message is triggered.

Reply:
"Please tell me what I should cook!" or so (I agree with Shadow)
tbh the original one is fine for me too